### PR TITLE
the one where publications have a short summary component added

### PR DESCRIPTION
**Pull request by sturobson**
This adds the `vf-summary--publication` component as well as a new helper class of `vf-et-al`.

The `vf-summary--publication` can be nested inside a `vf-box` where it takes the `vf-box` colours.

If the `vf-summary__author` list is truncated to a certain number of authors you will need to add `vf-et-al` to the `<p>` - `<p class="vf-summary__author | vf-et-al` - for it to add *et al* to the end of the list.

This _should_ cover all three use cases for publication summaries that are [here](https://gitlab.ebi.ac.uk/emblorg/backlog/issues/225). [here](https://gitlab.ebi.ac.uk/emblorg/backlog/issues/226), and [here](https://gitlab.ebi.ac.uk/emblorg/backlog/issues/227)

### CODE REVIEW

**Comment by sturobson** [comment]
I'm going to add what i've put in this PR description into the documentation.

File: gulpfile.js [hunk 1]
```diff
@@ -171,7 +171,6 @@ gulp.task('vf-css', function(done) {
             return 'Problem at file: ' + error.message;
           })
         )
-        .pipe(autoprefixer(autoprefixerOptions))
```
Comment (khawkins98) [inline gulpfile.js]: Was this intentional? I know we've discussed that dropping autoprefixer would likely be a good optimisation, but it's still in the package.json dependencies.

File: components/vf-summary/vf-summary.scss [hunk 2]
```diff
@@ -38,6 +62,13 @@
   color: set-color($vf-summary__date-color--text);
 }
 
+.vf-summary__source {
+  @include set-type($vf-summary__date-typography);
+
+  color: set-color($vf-summary__date-color--text);
+  font-style: italic;
```
Comment (khawkins98) [inline components/vf-summary/vf-summary.scss]: Feels weird using italic as we don't anywhere else, but I suppose that's a comment to direct to the design pattern/Mark.

File: components/vf-summary/vf-summary.scss [hunk 3]
```diff
@@ -12,6 +12,30 @@
     margin-bottom: 0;
   }
 
+  // vf-summary can also sit inside a vf-box - so we need to adjust some things to match
```
Comment (khawkins98) [inline components/vf-summary/vf-summary.scss]: Quite a few things might sit inside vf-box and i expect we'll have other things that might sit inside other things with background colours. When we have time, would be nice to have a method to support some kind of `vf-give-it-contrast` method.

File: components/vf-summary/vf-summary.scss (hunk 2 above)
Comment (sturobson) [reply components/vf-summary/vf-summary.scss]: Which I asked already ;)

File: components/vf-summary/vf-summary.scss (hunk 3 above)
Comment (sturobson) [reply components/vf-summary/vf-summary.scss]: Id rather work on something that either fixes this programmatically or adds specificity than sling yet another class onto something.

File: gulpfile.js (hunk 1 above)
Comment (sturobson) [reply gulpfile.js]: No. Unfortunately you can swap a PR to a draft PR. Which this still is.

Comment (sturobson) [reply gulpfile.js]: I'll put this back in and push

File: components/vf-summary/vf-summary.scss (hunk 3 above)
Comment (sturobson) [reply components/vf-summary/vf-summary.scss]: I think we can go with this now 'as is' and come back to it -- it'd be nice to look at component 'theming' in more detail for the next beta

File: components/vf-summary/vf-summary.scss (hunk 2 above)
Comment (sturobson) [reply components/vf-summary/vf-summary.scss]: checking this in Firefox using the fonts panel in the dev tools -- this is using the correct IBM Plex Sans Italic font file

**Comment by sturobson** [comment]
@khawkins98 I think we're good to do a first push of this now?

**Comment by khawkins98** [comment]
👍 Approved. Will leave you to merge when ready.

**Pull request merged**